pasteit/routes/functions.js

```diff
@@ -0,0 +1,47 @@
+var redis = require("redis");
+var client = redis.createClient();
+var uuid = require("uuid4");
+
+
+module.exports = {
+    newPaste : function(req) {
+        var newPaste = module.exports.createPaste(req);
+        var hash = uuid();
```

Comment:
Use only one "var" per function if possible, just put a comma after (req), and put hash on the next line.